app/src/sharedTest/java/org/oppia/app/faq/FAQFragmentTest.kt

```diff
@@ -0,0 +1,67 @@
+package org.oppia.app.faq
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.RecyclerViewActions
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Matchers.not
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.help.faq.FAQActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher
+
+@RunWith(AndroidJUnit4::class)
+class FAQFragmentTest {
+
+  var itemCount: Int = 9
```

Comment:
Avoid using `var` as much as possible and instead use `val`

`val itemCount: Int = 9`

Reply:
Done :)